### PR TITLE
Implement WORKDIR from the container image here

### PR DESCRIPTION
**Pull request by frenzymadness**
Fixes: https://github.com/fedora-python/tox-github-action/issues/16

### CODE REVIEW

**Comment by frenzymadness** [comment]
It does not work as expected right now and I'm not sure whether the problem is a possible collision of the name `workdir` or the tests here do not use the latest version of the action itself.

**Comment by frenzymadness** [comment]
I'm testing this in: https://github.com/frenzymadness/donors_registry/pull/190
And it seems to work well:
```
Complete!
Changing directory to migrations
py37 create: /github/workspace/.tox/py37
```

File: README.md [hunk 1]
```diff
@@ -46,12 +46,26 @@ groups or DNF options.
...
 [issue 8]: https://github.com/fedora-python/tox-github-action/issues/8
 
+If your application isn't in the root of your project, set `workdir`. The action
+will switch to the specified path before tox execution.
+
+```yaml
+- uses: fedora-python/tox-github-action
+  with:
+    tox_env: py38
+    workdir: "./python/"
```
Comment (hroncok) [inline README.md:56]: if the dot is not necessary I would not include it in the docs:

```suggestion
    workdir: "python/"
```

File: action.yaml [hunk 2]
```diff
@@ -9,12 +9,18 @@ inputs:
     required: false
     default: ''
     description: Space separated packages to install via dnf install (can contain options)
+  workdir:
+    required: false
+    default: ''
```
Comment (hroncok) [inline action.yaml:14]: I needed to check how this works. By default, this is set to an empty string and passed as an empty string to WORKDIR. On the other end: https://github.com/fedora-python/fedora-python-tox/blob/e5a0a5b81cdd42019f01c9671827dad0b9d1ae81/run_tests.sh#L19 WORKDIR is checked with `-z` which means check for emtynes. Alright then.

**Review by hroncok — APPROVED** [review]

**Comment by frenzymadness** [comment]
The proposed change has been applied and squashed.

**Pull request merged**